Services/HastebinService.cs

```diff
@@ -14,6 +14,18 @@ class HastebinService
 	{
 		private static readonly Regex _HasteKeyRegex = new Regex(@"{""key"":""(?<key>[a-z].*)""}", RegexOptions.Compiled);
 
+		private readonly string[] CodeBlockTypes = new string[]
+		{
+			"html",
+			"css",
+			"cs",
+			"dns",
+			"python",
+			"lua",
+			"http",
+			"markdown",
+		};
+
 		private readonly DiscordSocketClient _client;
 		private readonly LoggingService _loggingService;
```

Comment:
This field is unused, and even if it was couldn't `HastbinService` just extend `BaseService` to inherit `_loggingService`?